tests/AntDesign.Tests/Select/Renders/Select.Render.Tests.razor

```diff
@@ -45,62 +45,6 @@
         arrow.Should().NotBeNull();
     }
 
-    [Fact] //covers scenario from issue #1989
-    public async Task Render_select_dropdown_on_arrow_click()
```

Comment:
Why do you remove so many tests?

Reply:
BUnit cannot simulate the client events bound through JavaScript.

Reply:
Inblazor, using `@onclick:stopPromotion="@true"` on the child component does not prevent the parent component from responding to click events. Therefore, I replaced the event binding in blazer with binding through Js. 

Due to the above reasons, so this test is no longer executable.